modules/ROOT/attachments/admin-api.yaml

```diff
@@ -206,6 +206,191 @@ paths:
         200:
           description: Raft leadership transfer success
           content: {}
+  /v1/security/roles:
+    get:
+      parameters:
+        - description: Return only roles whose names match this substring
+          in: query
+          name: filter
+          required: false
+          schema:
+            type: string
+        - description: Return only roles which are assigned to this user
+          in: query
+          name: principal
+          required: false
+          schema:
+            type: string
+        - description: Applies to `user` parameter. Ignored if `user` is unset
```

Comment:
@oleiman Not sure which `user` parameter is being referred to here -- is it in a different endpoint?

Reply:
typo. should be `principal` 😕